offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -523,8 +523,7 @@ class OfferToPDFConverter implements OfferExporter {
             double taxRatio = determineTaxCost(affiliation.country, affiliation.getCategory())
             String taxPercentage = decimalFormat.format(taxRatio)
 
-            return """<div id="grid-table-footer">
-                                     <div class="col-10 cost-summary-field">Overheads (${overheadPercentage})</div>
+            return """<div class="col-10 cost-summary-field">Overheads (${overheadPercentage})</div>
```

Comment:
why not just replacing the cost values instead of removing and rewriting it where you need to copy the styles etc for the classes and the tags too.
I will not block the PR because of this, just something to think about.

Reply:
🚀